tests/tools/tflite_loader/src/tflite_loader.cc

```diff
@@ -45,7 +46,7 @@ const float DIFFERENCE_THRESHOLD = 10e-5;
   }
 
 // Read vector of floats from selected file
-std::vector<float> readData(const string &path)
+void readData(const string &path, uint8_t *dest)
```

Comment:
(optional) maybe add some assert checkes for `dest` ? caller can call with `nullptr`
also for below codes